### PR TITLE
Clarifications from Willem on what can be altered

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-dnsop-rfc8109bis.xml [hunk 1]
```diff
@@ -265,22 +265,30 @@ SHOULD be used for priming.</t>
...
-name server.  Note, however, that a validating resolver will not accept responses for signed TLDs from
-rogue root servers if they are different from the real responses because the resolver
-has a trust anchor for the root and the answers from the root are signed. Thus, if there
-is a machine-in-the-middle attack on the priming query, the results for a validating
-resolver for signed TLDs could be a denial of service, 
-or the attacker seeing queries while returning good answers,
-but not the resolver's accepting the bad responses; however, for unsigned TLDs, the attack would be successful.</t>
+<t>The root NS RRset is signed and can be validated by a DNSSEC validating resolver.
+At the time this document is published, the addresses for the names in the root NS RRset are in the "root-server.net" zone.
+All root servers are also authoritative for the "root-server.net" zone,
```
Comment (matthewhlarson) [inline draft-ietf-dnsop-rfc8109bis.xml]: Should be root-servers.net.

File: draft-ietf-dnsop-rfc8109bis.xml [hunk 2]
```diff
@@ -265,22 +265,30 @@ SHOULD be used for priming.</t>
...
-<t>A machine-in-the-middle attack on the priming query could direct a resolver to a rogue root
-name server.  Note, however, that a validating resolver will not accept responses for signed TLDs from
-rogue root servers if they are different from the real responses because the resolver
-has a trust anchor for the root and the answers from the root are signed. Thus, if there
-is a machine-in-the-middle attack on the priming query, the results for a validating
-resolver for signed TLDs could be a denial of service, 
-or the attacker seeing queries while returning good answers,
-but not the resolver's accepting the bad responses; however, for unsigned TLDs, the attack would be successful.</t>
+<t>The root NS RRset is signed and can be validated by a DNSSEC validating resolver.
+At the time this document is published, the addresses for the names in the root NS RRset are in the "root-server.net" zone.
```
Comment (matthewhlarson) [inline draft-ietf-dnsop-rfc8109bis.xml]: Should be root-servers.net.

**Review by matthewhlarson — CHANGES_REQUESTED** [review]

File: draft-ietf-dnsop-rfc8109bis.xml (hunk 1 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc8109bis.xml]: Fixed

File: draft-ietf-dnsop-rfc8109bis.xml (hunk 2 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc8109bis.xml]: Fixed

**Pull request merged**